README.md

```diff
@@ -1,9 +1,13 @@
-# th2 Sailfish Utils (3.8.0)
+# th2 Sailfish Utils (3.8.1)
 
 This library contains classes to convert messages from th2 to Sailfish format and vice versa. They are used in several th2 projects to reuse Sailfish features: message comparison, codec/connect implementations, etc.
 
 ## Release Notes
 
+### 3.8.1
+
++ Migrated sailfish-core version from `3.2.1622` to `3.2.1676`
```

Comment:
Please, add information about key changes in the new version (about the new parameter `keepResultGroupOrder` in `ComparatorSettings`)